spec/lib/dispatch-rider/message_spec.rb

```diff
@@ -31,28 +31,28 @@
 
   describe "#attributes" do
     it "should return the attributes hash of the message" do
-      message.attributes.should eq({:subject => 'test', :body => 'test_handler'})
+      expect(message.attributes).to eq({:subject => 'test', :body => 'test_handler'})
```

Comment:
Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.